commons-test/src/main/java/org/infinispan/commons/test/skip/StringLogAppender.java

```diff
@@ -0,0 +1,69 @@
+package org.infinispan.commons.test.skip;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Layout;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.appender.AbstractAppender;
+import org.apache.logging.log4j.core.config.AppenderRef;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+
+/**
+ * A Log4J Appender which stores logs in a StringBuilder
+ *
+ * @author Tristan Tarrant
+ * @since 9.2
+ */
+
+public class StringLogAppender extends AbstractAppender {
```

Comment:
Forgot to get rid of this?

Reply:
Didn't find a way to implement a thread id filter.

Reply:
ok, no problems!